benefits/core/templates/core/includes/media-item.html

```diff
@@ -1,12 +1,8 @@
-<li class="media d-flex align-items-stretch w-auto">
-    <div class="media-body">
-        {% block heading_wrapper %}
-            <{{ heading_tag|default:"h3" }} class="media-body--heading h3">
-            {% block heading %}
-            {% endblock heading %}
-            </{{ heading_tag|default:"h3" }}>
-        {% endblock heading_wrapper %}
-        {% block body %}
-        {% endblock body %}
-    </div>
+<li>
```

Comment:
Does it makes sense to keep these as `<li>`? 

Now that they are more "normal" sections with headers and copy... I wonder if we even need the list?

Reply:
Thoughts on renaming this file `eligibility-item`, to be very specific for where it's used? @angela-tran @thekaveman